screens/AccountCloseConfirmForm.js

```diff
@@ -0,0 +1,452 @@
+      	invoice: 'Invoice',
+      	others: 'Others',
+      	cancel: 'Cancel',
+      	remark: 'Closing Remark',
+      	totalCashIncome: 'Total Cash Income',
+      	totalCreditCardIncome: 'Total credit card Income',
+      	totalClosingAmount: 'Total Closing Amount',
+      	startingCash: 'Starting Cash',
+      	expectedCashRegister: 'Expected Cash In Register',
+      	actualCashInRegister: 'Actual Cash In Register',
+      	expectedCardAmtInRegister: 'Expected Card Amount In Register',
+      	actualCardAmtInRegister: 'Actual Card Amount In Register',
+      	totalInvoices: 'Total Number Of Invoices',
+      	deletedOrder: 'Total Number Of Orders Deleted',
+      	discounts: 'Total Amount Of Discount',
+      	serviceCharge: 'Total Service Charge',
+      	abort: 'Abort'
+      },
+      zh: {
+        Title: 'Closing Confirm-CH',
+        staff: '員工',
+        cash: 'Cash-CH',
+        card: 'Credit Card-CH',
+        confirm: 'Confirm-CH',
+        postClosingEntries: 'Post-Closing Entries-CH',
+        invoice: 'Invoice-CH',
+        others: 'Others-CH',
+        cancel: 'Cancel-CH',
+        remark:'Closing Remark-CH',
+        totalCashIncome: 'Total Cash Income-CH',
+        totalCreditCardIncome: 'Total credit card Income-CH',
+        totalClosingAmount: 'Total Closing Amount-CH',
+        startingCash: 'Starting Cash-CH',
+        expectedCashRegister: 'Expected Cash In Register-CH',
+        actualCashInRegister: 'Actual Cash In Register-CH',
+        expectedCardAmtInRegister: 'Expected Card Amount In Register-CH',
+        actualCardAmtInRegister: 'Actual Card Amount In Register-CH',
+        totalInvoices: 'Total Number Of Invoices-CH',
+        deletedOrder: 'Total Number Of Orders Deleted-CH',
+        discounts: 'Total Amount Of Discount-CH',
+        serviceCharge: 'Total Service Charge-CH',
+        abort: 'Abort-CH'
+      }
+    })
+  }
+
+  render() {
+    const { t } = this.context
+		const { handleSubmit, mostrecentShift, handleAbortCloseShift } = this.props
+
+      return (
+				<View>
+          {/* Post-Closing Entries */}
+          	<View style={styles.sectionBar}>             
+            	<View style={[styles.tableCellView, {flex: 4}]}>
+              	<TouchableOpacity>
+                	<Text style={styles.sectionBarTextSmall}>
+                  	{t('postClosingEntries')}
+                	</Text>
+              	</TouchableOpacity>
+            	</View>
+          	</View>
+
+          	<View style={[styles.container, styles.no_mgrTop]}>
+            	<View style={{flex: 3, justifyContent: 'center'}}>
+              	<View style={styles.fieldContainer}>
+               		<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                    	{t('totalCashIncome')}
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                	{
+										mostrecentShift.close.closingShiftReport !== null && 
```

Comment:
i suggest that instead of doing a lot of null checks in every section, you could define a data structure before the return statement to someting like following:

const closingShiftReport = {
totalByPaymentMethod: { 'CASH': { orderTotal: 0}, 'CARD': { orderTotal: 0} }}
}

if (mostrecentShift.close.closingShiftReport != null && mostrecentShift.close.closingShift.totalByPaymethod != null) {
   closingShiftReport.totalByPaymentMethod = mostrecentShift.close.closingShift.totalByPaymethod
}